tests/src/JIT/HardwareIntrinsics/Arm64/Simd.cs

```diff
@@ -222,8 +222,13 @@ class Program
             {
                 binOp(v,v);
             }
-            catch (PlatformNotSupportedException) // TODO-FIXME
+            catch (NotSupportedException e)
             {
+                if (!e.Message.Equals("Type is not supported."))
```

Comment:
This doesn't take localization into account, not sure if that's an issue.

Reply:
I was aware of that.  For initial testing it was sufficient to prove the message was set correctly.  

I'll drop it after #16102 merges.